src/types/institution.types.ts

```diff
@@ -1,46 +1,39 @@
-export interface InstitutionName {
+export interface RecursiveUnit {
   name: string;
-  language: string;
+  id: string;
+  subunits?: RecursiveUnit[];
 }
 
-export const emptyInstitutionNames: InstitutionName[] = [
-  {
-    name: '',
-    language: '',
-  },
-];
+export interface Subunit {
+  name: string;
+  id: string;
+}
 
-export interface Institution {
-  title?: string;
-  institutionNames: InstitutionName[];
-  acronym: string;
-  cristinInstitutionId: string;
-  country: string;
-  cristinUnitId: string;
+export interface FormikUnit {
```

Comment:
kunne hatt `extends Subunit` her, for å slippe å angi id og name flere ganger. Samme med `RecursiveUnit`

Reply:
renamer `Subunit` til `UnitBase` og extender 👍